apps/console/islands/nav.tsx

```diff
@@ -84,35 +78,13 @@ export const NavBar = () => {
         </li>
         <li className="cursor-not-allowed">
           <div className="flex items-center p-2 text-gray-400 hover:bg-sunset group ">
-            <svg
```

Comment:
Moving away from raw svgs here as ahead of time build occasionally struggle with these.